packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -250,6 +250,10 @@ export interface ExpoConfig {
    * Enable experimental features that may be unstable, unsupported, or removed without deprecation notices.
    */
   experiments?: {
+    /**
+     * Export a website relative to a subpath of a domain. The path will be prepended as-is to links to all bundled resources. Prefix the path with a `/` (recommended) to load all resources relative to the server root. If the path **does not** start with a `/` then resources will be loaded relative to the code that requests them, this could lead to unexpected behavior. Example '/subpath'. Defaults to '' (empty string).
```

Comment:
I think we should warn if the basePath is not absolute with an ENV to disable the warning. I'd rather help protect against typos than cater for an edge case